ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-driving web app framework",
+  "homepage": "https://www.celljs.org",
+  "keywords": [],
```

Comment:
Please add few relevant keywords here such as `self-driving, webapp, framework`. Any other keyword you think is relevant can also be added :)